package.json

```diff
@@ -5,17 +5,22 @@
   "bin": {
     "contentful-export": "./bin/contentful-export"
   },
-  "main": "index.js",
```

Comment:
Then we should IMHO rename that `run-contentful-export.js` into `index.js`

I see no reason to have another `index.js` on toplevel which is just im- and exporting another file.